src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -309,8 +325,8 @@ export class TeamlistComponent implements OnInit, OnDestroy {
     const temp = self.allTeams.slice();
     for (let i = 0; i < temp.length; i++) {
       temp[i]['isSelected'] = false;
-      if (self.selectedTeam['id'] === temp[i]['id']) {
-        temp[i]['isSelected'] = true;
+      if (self.otherSelectedTeam['id'] === temp[i]['id']) {
+        temp[i]['isSelectOtherselectedTeamed'] = true;
```

Comment:
I think the logic here doesn't seems right to me, also you are not using `isSelectOtherselectedTeamed`  key anywhere else. You are just assigning this key to `true` but I don't see for the `false`. I think you need to toggle this variable instead of setting directly to `true`.

Also please change the variable name

Reply:
@Sanji515 I think we don't need this variable as the `isSelect` will do the job here of `deSelecting` the previously selected team and `selecting` the new team.